pkg/commons/constants.go

```diff
@@ -0,0 +1,11 @@
+package commons
```

Comment:
We have tried to avoid doing something like if the reason was just to circumvent cyclic dependency. I'm okay duplicating this if multiple packages require just these constants.

Reply:
@kshlm @aravindavk what is your thought on this?

Reply:
I'm actually okay with doing this, as I don't want to duplicate code.

Reply:
@prashanthpai also its not just about avoiding the cyclic dependency. Rather its more to do with reuse of common constants. I personally agree with @kshlm here. @aravindavk ?

Reply:
commons package looks good to me, otherwise we will end up editing in multiple places if we have to change.

Reply:
Okay. Fair enough.

I'm afraid that we might end up putting lot more stuff (like struct definitions) under `common` [because of its name](https://blog.golang.org/package-names). I want to discourage that practice. IIUC, the `pkg` directory is meant to be home for reusable code like libraries which can be used by other projects too.

What are you thoughts about naming this package as `constants` because that's what it provides ?

Reply:
You a separate package `constants` parallel to `pkg`? I am fine with that.